migrations/900-remove-in-app.sql

```diff
@@ -0,0 +1,2 @@
+/* These will be recreated again automagically with the new data. */
+delete from inapp_products where simulate is not null;
```

Comment:
this should actually be ``... where stub = true`` (or however those messed up booleans in mysql work). This is because theoretically developers could kick their own products into simulation mode but we haven't added that feature yet.